app.go

```diff
@@ -69,6 +91,26 @@ func (f UpdateKey) MarshalJSON() ([]byte, error) {
 	})
 }
 
+func (app *App) getCurSor(idCursor string) ([]byte, error) {
+	type requestBody struct {
+		Id string `json:"id,string"`
+	}
+
+	req, err := app.newRequest("GET", "records/cursor", nil, "id="+idCursor)
+	if err != nil {
+		return nil, err
+	}
+	resGetCursor, err := app.do(req)
```

Comment:
Could you please describe variable name more clearly? 
And could you please also do it with others, if variable name is conflicted with golang syntax you can skip it. 
Thank you.